dev.c

```diff
@@ -232,9 +239,10 @@ void fuse_request_send_nowait(struct fuse_conn *fc, struct fuse_req *req)
 	}
 }
 
-static int request_pending(struct fuse_conn *fc)
+static bool request_pending(struct fuse_conn *fc)
 {
-	return fc->queue->r.read != fc->queue->r.write;
+	struct fuse_queue_cb *cb = &fc->queue->requests_cb;
+	return cb->r.read != cb->r.write;
```

Comment:
this change is fine. I see r.read and r.write as a ring pointers. So we have two independent rings, one on the kernel side receiving and informing data to px-storage (this is the "w" ring) and the other is the px-storage completing processing thats being informed through the "r" ring. Two independent rings right, but i see some correlation between "w" ring and "r" ring sometimes. havent understood that piece.